.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+on: [push]
+
+jobs:
+  build_and_test:
+    name: Build adn test on ${{ matrix.os }}
```

Comment:
Thanks for working on this! Just noticed a little typo here: the `adn` should be `and`.

Reply:
```suggestion
    name: Build and test on ${{ matrix.os }}
```

Reply:
Thanks 😘